src/test/java/gameframework/game/GameDataTest.java

```diff
@@ -0,0 +1,79 @@
+package gameframework.game;
+
+import static org.junit.Assert.*;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import gameframework.base.ObservableValue;
+
+public class GameDataTest {
+
+	GameData gd;
+	
+	@Before
+	public void init()	{
+		// GameConfiguration is 100 test-covered, thus trusted
```

Comment:
I disagree for several reasons:

1. 100% coverage does not mean bug free. There could still be bugs in `GameConfiguration`
2. Even if there are no bugs in `GameConfiguration`, there could be bugs in the integration of the 2 classes
3. Even if there are no integration bugs today, a developer could add bugs tomorrow

For all these reasons your comment does not justify the use of `GameConfiguration` in tests for `GameData`.